harvester/docs
# drone CI configuration and remove GitHub workflows

**problem**

Generating Harvester API documentation consistently fails when attempting to do so on Netlify and GitHub Actions.

**solution**

We will utilize Drone CI instead of GitHub Actions to address this issue, as building OpenAPI documents requires a machine with higher specifications.

**test plan**

1. install Drone CI on an AWS instance with a public IP.
2. Configure Drone and GitHub so that they can collaborate seamlessly.
3. Fork the repository at https://github.com/chrisho/docs and switch to the new-ci branch.
4. Make anywhere modifications and create a pull request.
5. Verify that the PR checks are successful and pass.
6. Merge the pull request and wait for Drone CI to complete its process.
7. Confirm that the gh-pages branch has been updated accordingly.
